### DISCUSSION TITLE
👀 Do you have a client of this version

### DISCUSSION THREAD

**Discussion by MarcHoog** [Hucaru/Valhalla#65] [Q&A] [answered]
Hey, amazing work you have done.  i wanted to play around with this repository as a hobby, however, do you have game client i can maybe borrow from this version? They are becoming harder and harder to find online.

**Comment by Hucaru** [comment, marked as answer]
You might be able to download from https://msdl.xyz/ or https://forum.ragezone.com/threads/maplestory-client-localhost-archive.1101897/ top of google search...

**Comment by MarcHoog** [comment]
The MSDL.xyz website wasn't working for me it always said no file attached here, and yeh i did overlook the rage forum post 😅 that's my bad. Thanks for answering though
